string bag_path
---
# Response (output) part
sensor_msgs/JointState joint_state
